include/EASTL/initializer_list.h

```diff
@@ -56,6 +56,14 @@
 			initializer_list() EA_NOEXCEPT  // EA_NOEXCEPT requires a recent version of EABase.  
 			  : mpArray(NULL), mArraySize(0) { }
 
+#if defined(EA_COMPILER_MSVC)
+			// MSVC generates constructor calls with two pointers instead of one pointer + size. The constructor is
+			// public.
+			// See: https://docs.microsoft.com/en-us/cpp/standard-library/initializer-list-class#initializer_list
+			initializer_list(const_iterator pFirst, const_iterator pLast) EA_NOEXCEPT
```

Comment:
Could/should this constructor be private like the one taking an iterator and size?

Reply:
No, it does not work. It is also public in MS STL: https://github.com/microsoft/STL/blob/2261f7edb760eb3fe0726187c818b796dc7ea798/stl/inc/initializer_list#L34

`1>C:\_GIT\.....(126,2): error C2248: 'std::initializer_list<eastl::pair<const Key,T>>::initializer_list': cannot access private member declared in class 'std::initializer_list<eastl::pair<const Key,T>>'

Reply:
Thanks for checking!